Dockerfile

```diff
@@ -25,7 +25,10 @@ COPY . .
 RUN export $(grep -v '^#' .env | xargs)
 
 # Change ubuntu timezone
-ENV TZ="America/Sao_Paulo"
+ENV TZ=America/Sao_Paulo
+RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
+RUN apt-get install tzdata -y
+RUN dpkg-reconfigure -f noninteractive tzdata
```

Comment:
**Suggestion:** The order of Dockerfile instructions matters. Installing packages should be done before setting the timezone. This is because the `tzdata` package, which is required for setting the timezone, might not be installed yet.
```suggestion
RUN apt-get update && apt-get install tzdata -y
ENV TZ=America/Sao_Paulo
RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
RUN dpkg-reconfigure -f noninteractive tzdata
```